src/main/java/com/imsweb/geocoder/GeocodeOutput.java

```diff
@@ -430,8 +430,8 @@ public void setPenaltyCodeSummary(String penaltyCodeSummary) {
                         result.setApiVersion(value(parts[1]));
                         result.setStatusCode(intValue(parts[2]));
 
-                        result.setLatitude(doubleValue(parts[3]));
-                        result.setLongitude(doubleValue(parts[4]));
+                        result.setLatitude(parts[3]);
+                        result.setLongitude(parts[4]);
```

Comment:
I think the `value` method should be called here, since we call it for all the other parts.